cypress/e2e/8-motions.js

```diff
@@ -178,9 +181,70 @@ describe('User can create motions via UAC', () => {
     });
   });
 
+  it('Claiming Stakes', () => {
+    cy.login();
+    cy.visit(`/colony/${colonyName}`);
+
+    // Get amount of staked tokens
+    cy.getBySel('tokenActivationButton', { timeout: 12000 }).click();
```

Comment:
I think that timeout should be `120000`, i need to update it to run. Please, modify.

Reply:
Thank you, this has been updated.